### ISSUE TITLE
compile with cygwin64 question on windows 10

### ISSUE THREAD

**Issue by saixmh** [OCamlPro/ocp-build#42]
I'm trying to use `opam` to install `ocp-build`.
Then it throw out the error message as below:

```
[ocp-build: make install] Command started
[ocp-build: sh] Command started
[ERROR] The installation of ocp-build failed at "sh -exc if [ -f
        'E:\\lingo\\.opam\\system/bin/ocp-build.asm' ]; then mv
        'E:\\lingo\\.opam\\system/bin/ocp-build.asm'
        'E:\\lingo\\.opam\\system/bin/ocp-build.exe' ; fi".
[ocp-build: rm] Command started
...
```

Do you have any idea?  Thx.

**Comment by lefessan** [comment]
You should probably ask @fdopen as he is the maintainer of this package (we maintain only the non-mingw64 version of `ocp-build`).

However, I suspect this error is linked to the fact of using the `system` compiler. Do you have read/write access to `E:\\lingo\\.opam\\system/bin/` ?

**Comment by fdopen** [comment]
Indeed. If you use my repository, don't use the system compiler. Only the switches maintained by this repo (e.g. `4.02.3+mingw64`) set the right (environment) variables.
Also the other binaries distributed by eg. msprotz (like flexlink.exe, etc.), are not fully compatible with the repo. Use the bootstrap binaries linked under https://github.com/fdopen/opam-repository-mingw

Beside that, ocp-build (as packaged there) is only in a half usable state. It works well enough to compile packages like ocp-indent. But if you intended to use it yourself and your projects are more complex, you will soon discover the deficiencies.

**Comment by saixmh** [comment]
@lefessan @fdopen 
Thanks for your suggestions. It's already work fine.

**Comment by lefessan** [comment]
@fdopen Could you give us more info on the deficiencies you have noticed ? We use ocp-build directly with ocpwin, and except a quite verbose output, we had no other problem.
